planty/users/forms.py

```diff
@@ -0,0 +1,18 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+
+
+class UserRegisterForm(UserCreationForm):
+    email = forms.EmailField()
+
+    class Meta:
+        model = User
+        fields = ['username', 'email', 'password1', 'password2']
+
+    # Hide help text in the form
+    def __init__(self, *args, **kwargs):
+        super(UserCreationForm, self).__init__(*args, **kwargs)
+
+        for fieldname in ['username',  'email', 'password1', 'password2']:
```

Comment:
I think it would work if you reference the `fields` attribute from the `Meta` class above:
```python
for fieldname in Meta.fields:
    ...
```